webapp/src/router.ts

```diff
@@ -220,6 +220,12 @@ export default function router(server, app) {
 		csrfMiddleware,
 		accountController.switchTeam
 	);
+	accountRouter.post(
+		'/onboarded',
```

Comment:
This route needs to have middlewares to do authentication, see the middlewares for /logout or /switch above

Reply:
i dont think we need all middlewares except for 

		unauthedMiddlewareChain,
		checkSession,

Reply:
We also need csrfMiddleware, and to add a csrf token to the request. You can get the csrf token from the account context, take a look at one of the forms e.g. AgentForm to see an example :slightly_smiling_face:

Reply:
csrfmiddleware added